vault/dendron.tutorial.conclusion.md

```diff
@@ -11,23 +11,62 @@ nav_order: 4
 
-#### Join the community of Dendrologists
+## Join the Dendron Community
 
-Don't forget - join us on [Discord](https://discord.com/invite/AE3NRw9) to ask questions, suggest new features, or to discuss note-taking methodologies with fellow Dendrologists.
+![[dendron://dendron.dendron-site/community]]
 
-^discord-block-ref
+Check out our [[Case Studies|dendron://dendron.dendron-site/dendron.case-studies]] and [[Workflows|dendron://dendron.dendron-site/dendron.guides.workflows]] to see more ways Dendron is being used to organize knowledge.
 
-#### Explore the full potential of Dendron
+## Publishing
 
-There is much more that Dendron offers. When you're ready, learn more about additional features that Dendron offers:
+Dendron isn't just for taking notes. It can be used to share your thoughts and documentation with others. One of those ways is with **publishing**. The [Dendron Documentation](https://wiki.dendron.so) and [Dendron Blog](https://blog.dendron.so) are both published this way.
 
-- [ ] Import your existing notes from other sources like Obsidian, etc. with [Pods](https://wiki.dendron.so/notes/66727a39-d0a7-449b-a10d-f6c438185d7f.html)
-- [ ] [Publish](https://wiki.dendron.so/notes/861e4e48-dcc5-4813-a695-8940ba6e64d3.html) your notes to a website with just a few clicks. Our [Dendron Wiki Site](https://wiki.dendron.so/) is created in this way!
-- [ ] Check out our [Case Studies](https://wiki.dendron.so/notes/34ee4bcf-60e9-4031-a4c0-26113b5acb80.html) and [Workflows](https://wiki.dendron.so/notes/9313b845-d9bf-42c9-aad1-0da34794ce26.html) to see various ways Dendron is being used to organize knowledge.
-- [ ] Learn about more [Commands](https://wiki.dendron.so/notes/eea2b078-1acc-4071-a14e-18299fc28f47.html)
+If you want to learn more about [[publishing your notes online|dendron://dendron.dendron-site/dendron.topic.publish.quickstart]], you can take a look at how to publish with example guides for [[Netlify|dendron.topic.publish.cook.netlify]] or [[GitHub Pages|dendron.topic.publish.cook.github]].
 
-- [ ] Check out our [Seed Bank](https://wiki.dendron.so/notes/6ff8cbb6-e4b8-449b-a967-277b76e4ecef.html) to pull down documentation for public topics.
+![[dendron://dendron.dendron-site/asset.preview#publishing-nextjs-site-preview,1:#*]]
 
+## Explore the full potential of Dendron
 
-### Other Resources
-- Fellow dendrologist [Ian Jones](https://github.com/theianjones) made a fantastic video series about getting started with Dendron [here](https://egghead.io/courses/build-a-personal-knowledge-management-system-with-dendron-b24b)
\ No newline at end of file
+There is much more that Dendron offers. When you're ready, learn more about additional features with this _Choose Your Own Adventure_ list of learnings. Each section represents and area of a map that goes deeper into the world of Dendron.
```

Comment:
update, this is less organized in difficulty, not topics. 

can we change to:

> There is much more that Dendron offers. When you're ready, learn more about our additional features

We can then explain the various sections in a sentence below each header (see below commnents)